CHANGELOG.md

```diff
@@ -40,11 +40,7 @@
   written in Go and used by the static website generator
   [Hugo](https://gohugo.io/). (@vanillajonathan, #3597)
 - [scc](https://github.com/boyter/scc), a source lines of code counter now has
-  support for `.prql` files. (@vanillajonathan)
-- [gcloc](https://github.com/JoaoDanielRufino/gcloc) a source lines of code
-  counter now has support for `.prql` files. (@vanillajonathan)
-- [cloc](https://github.com/AlDanial/cloc) a source lines of code counter now
-  has support for `.prql` files. (@AlDanial)
```

Comment:
Did we intend to remove these?

Reply:
No, we do not intend to remove those. Good spotted, not sure what went wrong.